internal/services/containers/kubernetes_cluster_network_resource_test.go

```diff
@@ -26,6 +27,28 @@ func TestAccKubernetesCluster_advancedNetworkingKubenet(t *testing.T) {
 		data.ImportStep(),
 	})
 }
+func TestAccKubernetesCluster_advancedNetworkingKubenetWithIPVersions(t *testing.T) {
```

Comment:
Because `ip_versions` is a force new property, so better split this testcase into 2 cases.